files/build_templates/init_cfg.json.j2

```diff
@@ -183,5 +183,11 @@
             "admin_state": "enabled",
             "vrf": "default"
         }
+    },
+    "STORMOND_CONFIG": {
```

Comment:
>STORMOND_CONFIG [](http://example.com/codeflow?start=5&length=15)

What is the motivation to change init_cfg.json? What will happen if we do not change?

If the motivation is about default value, I suggest define default value in Yang model, and let the consumer side code to interpret the missing field as default value.

Reply:
Hi @qiluo-msft: the motivation is to address the feedback I received from community - to make the storage monitoring daemon's polling intervals configurable by end user.

Reply:
I have left the fields empty and added default param to the YANG model. @qiluo-msft

Reply:
Sorry, I mean remove the `STORMOND_CONFIG`, and you can still use default value in code.

Reply:
Got it, will do. Thanks @qiluo-msft !

Reply:
still applicable.

Reply:
Hi @qiluo-msft , correct me if I'm wrong but this is an outdated commit -- if you see here I have now only changed 4 files in this PR, all in sonic-yang-models.

Reply:
<img width="601" alt="image" src="https://github.com/sonic-net/sonic-buildimage/assets/93744978/49807c6c-b78a-4fd5-91de-fe0d4efd900b">